frontend/src/app/home/home.component.ts

```diff
@@ -22,32 +28,32 @@ export class HomeComponent implements OnInit {
   }
 
   generateKeypair(): void {
-    this.midware.generateKeypair().subscribe(keypair => {
+    this.middleware.generateKeypair().subscribe(keypair => {
       console.log(keypair);
       this.midObj = { ...this.midObj, keypair };
     });
   }
   get(): void {
-    this.midware.checkGet().subscribe(midware => {
-      this.midObj = midware;
+    this.middleware.checkGet().subscribe(middleware => {
+      this.midObj = middleware;
```

Comment:
Does setting `this.midObj` this way overwrite existing properties?

Reply:
@dsifford Yes - but this will all be blown away in #12 as these were just for testing connection to middleware.